about_me.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+        <head>
+                <link rel="stylesheet" type="text/css" href="about_me.css">
+        </head>
+<body>
+        <h3><i>About Me </i>| <a href="portfolio.html">Portfolio</a></h3>
+        <h1>Sandy Vasquez</h1>
+        <hr>
+        <h4>
+            I Enjoy Life As a Developer!  
+        </h4>
+        <p>
+                I'm Sandy Vasquez, a developer based in NYC.
+                <img src="https://placeimg.com/220/210/tech" alt"technical image" >
+        </p>
+        <hr>
+
+        <nav>
+                        <ul>
```

Comment:
@sandyvasquez great use of indentation here but don't forget to be consistent. this <ul> tag does not need that much indentation.